Plugins/NativeEngine/Source/ShaderCompilerCommon.cpp

```diff
@@ -172,7 +172,7 @@ namespace Babylon::ShaderCompilerCommon
             const auto& compiler = *vertexShaderInfo.Compiler;
             const spirv_cross::ShaderResources resources = compiler.get_shader_resources();
             auto uniformsInfo = CollectNonSamplerUniforms(*vertexShaderInfo.Parser, compiler);
-#if (BGFX_CONFIG_RENDERER_METAL)
+#if __APPLE__
```

Comment:
why is this better than the previous define?

Reply:
Not better, but more consistent with checks in other cpp like ShaderCompilerTraversers.

Reply:
Apple in theory can use OpenGL even though it's deprecated, so maybe the other define is better than checking __APPLE__